batch/2.0/_index.md

```diff
@@ -1,23 +1,25 @@
 ---
-title: "Jakarta Batch 2.0 (under development)"
-date: 2020-01-15
+title: "Jakarta Batch 2.0 (milestone)"
+date: 2020-06-09
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Batch specifies a Java API plus an XML-based job specification language (JSL), which lets you compose batch jobs in XML from reusable Java application artifacts and conveniently parameterize different executions of a single job.
 
 * [Jakarta Batch 2.0 Release Record]()
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Batch 2.0 Specification Document]() (PDF)
-* [Jakarta Batch 2.0 Specification Document]() (HTML)
+* [Jakarta Batch 2.0 Specification Document](./batch-spec-2.0-draft.pdf) (PDF)
```

Comment:
The actual files that you posted to this PR do not have the "-draft" suffix...  Thus, these links do not work.

Reply:
Done

Reply:
:-) I would prefer that you rename the actual files to have the -draft suffix rather than dropping it from the reference.  Otherwise, some consumers could confuse the downloaded files as being "final" based on the name.  Sure, you can look inside the docs to see that it's still Draft, but it would be clearer if the file name would indicate it's not final.

Reply:
Thanks.